packages/api/cms-api/src/dam/files/dto/file.input.ts

```diff
@@ -58,16 +58,16 @@ export class LicenseInput {
     author?: string;
 
     @Field(() => Date, { nullable: true })
-    @Transform(({ value }) => new Date(value))
+    @Transform(({ value }) => (value ? new Date(value) : undefined))
     @IsOptional()
     @IsDate()
-    durationFrom?: Date;
+    durationFrom?: Date | null;
```

Comment:
```suggestion
    durationFrom?: Date;
```

Since it's now always transformed to undefined, it can never be null